src/transforms/group.js

```diff
@@ -201,10 +202,16 @@ const reduceCount = {
   }
 };
 
+function sumOrCount(I, V) {
+  return typeof V.find(v => v != null) === "number"
```

Comment:
This could lead to inconsistent behavior across series (and facets). Would it be possible to do this earlier, on the first non-null value in the entire channel, as we do for other forms of type inference?

Reply:
I think that's what I'm doing here: V is the entire channel. What I don't like is I'm doing the same for each series, without caching.

Reply:
Ah, thanks for the clarification.